bookwyrm/views/admin/user_admin.py

```diff
@@ -117,3 +118,37 @@ def post(self, request, user_id):
         user = get_object_or_404(models.User, id=user_id)
         user.reactivate()
         return redirect("settings-user", user.id)
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    permission_required("bookwyrm.moderate_user", raise_exception=True),
+    name="dispatch",
+)
+class ForcePasswordResetAdmin(View):
+    """Require a group of users to reset their passwords"""
+
+    def get(self, request):
+        """user view"""
+        data = {"groups": Group.objects.all()}
+        return TemplateResponse(
+            request, "settings/users/force_password_reset.html", data
+        )
+
+    def post(self, request):
+        """force password reset and log out groups of users"""
+        group = request.POST.get("group")
+        user_queryset = models.User.objects.filter(local=True)
+        if group:
+            user_queryset = user_queryset.filter(group__name=group)
```

Comment:
Oops. Should be:
```py
user_queryset = user_queryset.filter(groups__name__contains=group)
```

Reply:
interestingly, `groups__name=group` does work -- Django querysets are flexible with lists.